relay-server/src/envelope.rs

```diff
@@ -334,6 +334,17 @@ pub struct ItemHeaders {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     filename: Option<String>,
 
+    /// Indicates that this item is being rate limited.
+    ///
+    /// By default, rate limited items are immediately removed from Envelopes. For processing,
+    /// native crash reports still need to be retained. These attachments are marked with the
+    /// `rate_limited` header, which signals to the processing pipeline that the attachment should
+    /// not be persisted after processing.
+    ///
+    /// NOTE: This is internal-only and not exposed into the Envelope.
```

Comment:
What does that mean?

Reply:
This means that we apply `#[serde(skip)]`, so this is never deserialized from an incoming envelope, and it's also never serialized into a forwarded envelope.